phpBB/phpbb/textformatter/s9e/quote_helper.php

```diff
@@ -53,6 +54,7 @@ public function __construct(\phpbb\user $user, $root_path, $php_ext)
 	public function inject_metadata($xml)
 	{
 		$post_url = $this->post_url;
+		$msg_url = $this->msg_url;
```

Comment:
No need to declare a local variable here. The reason those variables exist is because closures couldn't access non-public properties in PHP 5.3. Now that it's not required anymore, I'd be in favor of removing those variables, removing the `use` statement from the closure and access the properties directly from the closure.